packages/serverpod/test/database/columns/column_bigint_test.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:serverpod/database.dart';
```

Comment:
There is also an integration test in `tests/serverpod_test_server/test_integration/column_operations/column_duration_test.dart` that should be added for `BigInt` as well.

Reply:
Added integration test.